x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/inference/rescorer/InferenceRescorerFeature.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+package org.elasticsearch.xpack.ml.inference.rescorer;
+
+import org.elasticsearch.common.util.FeatureFlag;
+
+public class InferenceRescorerFeature {
```

Comment:
I think I did feature flag stuff correctly. @mayya-sharipova could you confirm? I think you have done this recently.

Reply:
That's correct, Ben!